Backends/common/events.lisp

```diff
@@ -1,176 +0,0 @@
-(in-package #:climi)
-
-(defclass standard-event-port-mixin ()
```

Comment:
Is it conceivable that a port is not a `basic-port` but wants this behavior?

Reply:
according to the spec all ports should inherit from basic-port (otherwise all protocols involving port need to be implemented by the vendor). this mixin is an abstraction bloat complicating the mental image of the software workings, basic-port provides the "standard" CLIM behavior.

Reply:
OK.